docs/api-docs/store-management/shipping/shipping-provider-api.md

```diff
@@ -47,31 +47,31 @@ Please include:
 * Email
 * Description of the app
 
-To get your app ID, create an app in [Developer Tools](https://devtools.bigcommerce.com/) and fill out the information on [Step 3 Technical](https://developer.bigcommerce.com/api-docs/partner/app-store-approval-requirements). In the URL the app will have a unique ID. This is what is sent in exchange for a carrier ID which can be used to test the app.
+To get your app ID, create an app in [Developer Tools](https://devtools.bigcommerce.com/), and fill out the information on [Step 3 Technical](https://developer.bigcommerce.com/api-docs/apps/guide/requirements#technical). In the URL, the app will have a unique ID. Send the unique ID in exchange for a carrier ID to test the app.

```

Comment:
The link for Step 3 Technical takes you to App Store Approval Requirements.

Reply:
I changed the link. 
I think this link will work.
https://developer.bigcommerce.com/api-docs/apps/guide/publishing#add-technical-information